src/TestInfrastructure/VsObjectModelMocks/ProjectMock.cs

```diff
@@ -294,9 +294,12 @@ int IVsProject.AddItem(uint itemidLoc, VSADDITEMOPERATION dwAddItemOperation, st
         int IVsBuildPropertyStorage.GetPropertyValue(string pszPropName, string pszConfigName, uint storage, out string pbstrPropValue)
         {
             pbstrPropValue = null;
-            return this.buildProperties.TryGetValue(pszPropName, out pbstrPropValue)
-                ? VSConstants.S_OK
-                : VSConstants.E_FAIL;
+            if (this.buildProperties.ContainsKey(pszPropName))
```

Comment:
No. [We fire and forget (+ assert)](https://github.com/SonarSource-VisualStudio/sonarlint-visualstudio/pull/68/files#diff-6db57c7951ddca736f47debb5d8453d6R269) - we treat failure to get the property the same as the property not existing. There's no retry logic, or anything else we can do (or want to do).

Anything else would be us testing that the _actual_ `IVsBuildPropertyStorage` implementer which we won't do.